scalability_and_performance/understanding-node-observability-operator.adoc

```diff
@@ -0,0 +1,81 @@
+:_content-type: ASSEMBLY
+[id="using-node-observability-operator"]
+= Understanding the Node Observability Operator
+include::_attributes/common-attributes.adoc[]
+:context: node-observability-operator
+
+toc::[]
+
+Node Observability Operator is an optional add-on Operator that provides a link:https://cri-o.io/#what-is-cri-o[CRI-O] and link:https://kubernetes.io/docs/concepts/overview/components/#kubelet[kubelet] profiling on selected worker nodes.
+
+== Key features
+* Trigger a CRIO-O and kublet profiling.
+* You can deploy Node Observability Agents on selected nodes.
+* Users can access the profiling data files for analysis.
+
+== Installing the Node Observability Operator using the web console
+The Node Observability Operator is not installed in {product-title} by default. You can install the Node Observability Operator by using the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the OpenShift Container Platform web console.
+. Navigate to *Operators* → *OperatorHub*.
+. Enter `node-observability-operator` into the filter box to locate the Node Observability Operator.
+. Click *Install*.
+. On the *Install Operator* page:
+.. *Update channel*: Select *alpha*.
+.. *Installation mode*: Select *A specific namespace on the cluster*.
+.. *Installed Namespace*: Choose *node-observability-operator* from the list.
+.. *Update approval*: Select *Automatic*.
+.. Click *Install*.
+
+== Deploying the Node Observability Operator
+Before you can deploy the Node Observability Operator, you should create a `NodeObservability` custom resource (CR).
+
+.Prerequisites
+* The Node Observability Operator has been installed.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role:
++
+[source,terminal]
+----
+$ oc login -u kubeadmin https://<HOSTNAME>:6443
+----
+
+. Create a new project named `node-observability-operator`:
++
+[source,terminal]
+----
+$ oc project node-observability-operator
+----
+
+. Create a custom resource file named `nodeobservability.yaml` that contains the following text:
++
+[source,yaml]
+----
+    apiVersion: nodeobservability.olm.openshift.io/v1alpha1
+    kind: NodeObservability
+    metadata:
+      name: cluster <1>
+    spec:
+      labels:
+        "node-role.kubernetes.io/worker": ""
+      image: "brew.registry.redhat.io/rh-osbs/node-observability-agent:0.1.0-3"
```

Comment:
`image` was removed from the spec starting https://github.com/openshift/node-observability-operator/pull/69 
But you're right, we need to update the readme in https://github.com/openshift/node-observability-operator/tree/main/docs#prepare-to-run-profiling-queries
In the meantime, you can use example from : https://github.com/openshift/node-observability-operator/blob/main/config/samples/nodeobservability_v1alpha1_nodeobservability-all.yaml

Reply:
Fixed.